pvlib/irradiance.py

```diff
@@ -68,19 +73,24 @@ def get_extra_radiation(datetime_or_doy, solar_constant=1366.1,
     .. [1] M. Reno, C. Hansen, and J. Stein, "Global Horizontal Irradiance
        Clear Sky Models: Implementation and Analysis", Sandia National
        Laboratories, SAND2012-2389, 2012.
+       :doi:`10.2172/1039404`
 
-    .. [2] http://solardata.uoregon.edu/SolarRadiationBasics.html, Eqs.
-       SR1 and SR2
+    .. [2] J. A. Duffie, W. A. Beckman, N. Blair, "Solar Radiation", in Solar
+       Engineering of Thermal Processes, Photovoltaics and Wind, 5th ed,
+       New York, USA: J. Wiley and Sons, 2020, pp. 3-44.
+       :doi:`10.1002/9781119540328`
 
-    .. [3] Partridge, G. W. and Platt, C. M. R. 1976. Radiative Processes
-       in Meteorology and Climatology.
+    .. [3] J. W. Spencer, "Fourier series representation of the sun," Search,
+       vol. 2, p. 172, 1971.
 
-    .. [4] Duffie, J. A. and Beckman, W. A. 1991. Solar Engineering of
-       Thermal Processes, 2nd edn. J. Wiley and Sons, New York.
+    .. [4] R. G. Allen, Environmental, and E. Water Resources institute. Task
+       Committee on Standardization of Reference, The ASCE standardized
+       reference evapotranspiration equation. Reston, Va.: American Society of
+       Civil Engineers, 2005. :doi:`10.1061/9780784408056`
```

Comment:
Let's fix this one also. The reference is a book with editors, rather than authors. I can't view a copy to identify which chapter contains the extraterrestrial radiation equation, so we'll reference the whole book.

```
[4] R. G. Allen et al., Eds. The ASCE standardized
       reference evapotranspiration equation. Reston, Va.: American Society of
       Civil Engineers, 2005. :doi:`10.1061/9780784408056`
```